libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/room/MatrixRoom.kt

```diff
@@ -98,7 +98,9 @@ interface MatrixRoom : Closeable {
 
     val syncUpdateFlow: StateFlow<Long>
 
-    val timeline: MatrixTimeline
+    val liveTimeline: Timeline
```

Comment:
Maybe add docs for this and `timelineFocusedOnEvent`?

Reply:
Yes, added. Also I believe we will sort of rework this API to distinguish the live timeline and the detached timeline.
We should not use the detached timeline to send an event for instance.